programs/create-genesis/ee_genesis/event_engine_genesis.cpp

```diff
@@ -129,6 +201,7 @@ static abi_def create_accounts_abi() {
             {"balance", "asset"},
             {"balance_in_sys", "asset"},
             {"vesting_shares", "asset"},
+            {"received_vesting_shares", "asset"},
```

Comment:
both delegated and received vests can be obtained from given delegation ops. is it ok to add add only one field here?
btw, there are expiring (returning) delegations, they are not part of ee-genesis, is it ok?